SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslator.cpp

```diff
@@ -280,6 +279,7 @@ void SpatialVirtualWorkerTranslator::UpdateMapping(VirtualWorkerId Id, PhysicalW
 		bIsReady = true;
 
 		// Tell the strategy about the local virtual worker id.
+		check(LoadBalanceStrategy != nullptr);
```

Comment:
Isn't `.IsValid()` check sufficient?

Reply:
It could be, however this check was just moved from the top of this file. This is done because this class is now being instantiated by the client which won't have a LoadBalanceStrategy, but we still want to do the check on servers - hence moving it here.

Reply:
oh I see what you mean haha, im being blind, yes it should be